signalbot/storage.py

```diff
@@ -43,23 +60,29 @@ def save(self, key: str, object: Any):
 
 class RedisStorage(Storage):
     def __init__(self, host, port):
-        self._redis = redis.Redis(host=host, port=port, db=0)
+        self.redis = redis.Redis(host=host, port=port, db=0)
 
     def exists(self, key: str) -> bool:
-        return self._redis.exists(key)
+        return self.redis.exists(key)
 
     def read(self, key: str) -> Any:
         try:
-            result_bytes = self._redis.get(key)
+            result_bytes = self.redis.get(key)
             result_str = result_bytes.decode("utf-8")
             result_dict = json.loads(result_str)
             return result_dict
         except Exception as e:
             raise StorageError(f"Redis load failed: {e}")
 
+    def delete(self, *keys: str) -> int:
```

Comment:
Totally optional but how do you feel about adding some unit testing for the new function?